desktop/appcontainer/AboutDialog.ts

```diff
@@ -42,10 +42,25 @@ export const aboutDialog = hoistCmp.factory({
                     item: model.getTable()
                 }),
                 toolbar(
+                    button({
+                        text: 'Send Client Health Report',
+                        icon: Icon.health(),
+                        omit: !XH.trackService.enabled,
+                        onClick: async () => {
+                            try {
+                                XH.clientHealthService.sendReport();
+                                await XH.trackService.pushPendingAsync();
+                                XH.successToast('Client health report successfully submitted.');
+                            } catch (e) {
+                                XH.handleException('Error sending client health report', e);
+                            }
```

Comment:
Thought about bundling toast, push, error handling into options on `sendReport` but didn't really seem worth it. 

I did think it was worth the extra complexity to do both of those things, however, to ensure the report gets submitted right away, assuming some kind of interactive troubleshooting is likely

Reply:
👍🏻